repos/keg-components/src/components/button/button.js

```diff
@@ -18,7 +18,7 @@ import { useThemeTypeAsClass } from 'KegTypeAsClass'
  * @returns {React Component|Object|Array}
  */
 const getChildren = (Children, styles = {}) => {
-  return renderFromType(Children, { style: styles.content }, Text)
+  return renderFromType(Children, { style: styles.content, selectable: false }, Text)
```

Comment:
Not blocking, but I think we could make this a prop in `button.js` so that the consumer could still make it selectable if they wanted to.

Reply:
well if the children/content isn't a string its selectable by default
the text on this one would be selectable
```
      <Button
        themePath='button.contained.primary'
        styles={{ main: buttonStyle }}
        onClick={action('Button Clicked!')}
        
      >
          <Text>Hello</Text>
      </Button>
```

which i think makes sense if a consumer use the `content` prop just for a simple text. they will have the expectations that it works a certain way and if they want to customize it they just pass in their own children component

Reply:
Hmm, I like the idea of making it a prop. That way the can override the default, but don't need to.
@daniel112 - It would still work in the way you described, but also give the option to override.

Reply:
updated